docs/sdkx_ios/support-tools.mdx

```diff
@@ -166,23 +166,65 @@ You can use the API `showSingleFAQ:with:config:` to show a single FAQ.
 
 </TabItem>
 <TabItem value="Swift Example" label="Swift Example">
 
 |              |                                                                                                                                                                                                                                                                                                                                                                                      |
 | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Definition   | <code>showSingleFAQ(with:config:)</code>                                                                                                                                                                                                                                                                                                                                             |
-| Example Code | <code> Helpshift.showSingleFAQ(faqId, with: self, config: configDictionary) </code> // where <code>faqId</code> is the <code>publishId</code> of that particular FAQ, <code>self</code> is the view controller you're calling Helpshift from and <code>configDictionary</code> is a <code>Dictionary</code> that represents the configuration that needs to be set to the FAQ screen |
+| Definition   | <code>showSingleFAQ(_:withConfig:)</code>                                                                                                                                                                                                                                                                                                                                             |
+| Example Code | <code>Helpshift.showSingleFAQ(faqId, withConfig: configDictionary)</code> where <code>faqId</code> is the <code>publishId</code> of that particular FAQ and <code>configDictionary</code> is a <code>Dictionary</code> that represents the configuration that needs to be set to the FAQ screen |
 
 </TabItem>
 
 </Tabs>
 
 Supports these [API Options](/sdkx_ios/sdk-configuration/).
 
+## Closing an open SDK session
+
+You can use the API `closeSession` to close an active SDK session. Calling this API will close any open SDK screens and invoke the corresponding [lifecycle delegate methods](/sdkx_ios/delegates/#session-delegates). If no SDK session is active, calling this API will have no effect.
+
+<Tabs>
+<TabItem value="Objective-C Example" label="Objective-C Example">
```

Comment:
Don't think we are using "Example" in value and label elsewhere. Please confirm if it is needed.

Reply:
We have it for all tab item labels on this page. So I kept it to maintain consistency.